pom.xml

```diff
@@ -350,5 +350,29 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
-</project>
+
+   <profiles>
+      <profile>
```

Comment:
You can limit to run a single test with mvn -Dtest=GitSCMTest clean install.  No need for a profile dedicated to run a single test.